.gitignore

```diff
@@ -59,3 +59,9 @@ typings/
 
 # next.js build output
 .next
+
+#Mac OS Finder DS_Store files
+.DS_Store
+
+#JetBrains IDEa config
+.idea/
```

Comment:
You'll probably want to ignore log files and other IDEs, eg:
npm-debug.log*
yarn-debug.log*
yarn-error.log*
.vscode